hack/build-helm-image.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/env bash
+
+# Copyright 2023 The KubeStellar Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Purpose: build a helm image and push it to the registry.
+
+# Usage: $0 [--version|-v release] [--registry registry] [--platform platforms] [-X]
+# Working directory does not matter.
+
+set -e
+
+helm_version="" # ==> latest
+helm_folder=helm
+registry=quay.io/kubestellar
+platform=linux/amd64,linux/arm64,linux/ppc64le
```

Comment:
Do we have OCM for linux/ppc64le  ?

Reply:
This is not for OCM but yes, I have build for linux/amd64, linux/arm64, and linux/ppc64le as you can see in quay.io.

Reply:
FYI - open cluster management only has  images for linux/amd64 and linux/arm64 - see for example https://quay.io/repository/open-cluster-management/work?tab=tags (on the agent side) and https://quay.io/repository/open-cluster-management/placement?tab=tags (on the hub side). So even if we build for other archs and we use OCM that will not help.

Reply:
@francostellari just to confirm,  If I don't pass any flags it will just build with the defaults in lines 24-27, right ?